### PR TITLE
Update workouts extension

### PR DESCRIPTION
**Pull request by diannemcewan**
## Checklist

- [ ] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ ] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by codesjedi** [comment]
hi @diannemcewan, some checks were not successful, can you take a quick look please?

File: extensions/workouts/src/api/client.ts [hunk 1]
```diff
@@ -194,3 +196,43 @@ export const getClubActivities = async (clubId: string, page = 1, pageSize = PAG
     throw new Error(error);
   }
 };
+
+export const createActivity = async (activityValues: StravaManualActivity) => {
+  console.log("activityValues");
```
Comment (samuelkraft) [inline extensions/workouts/src/api/client.ts:201]: remove logs

**Review by samuelkraft — APPROVED** [review]
Works great! 👏

It'd be nice if there was a way to visit the created activity on Strava (or inside the extension) after it has been created and for the form to either close or clear the values.

**Comment by samuelkraft** [comment]
Here's a command icon @diannemcewan 

add-workout

File: extensions/workouts/src/api/client.ts (hunk 1 above)
Comment (diannemcewan) [reply extensions/workouts/src/api/client.ts:201]: I agree on the return action. I spent some time on trying to clear the form values, but could not find a nice way to get I working, so I thought I would leave it our for now.

Comment (samuelkraft) [reply extensions/workouts/src/api/client.ts]: you could probably make the form controlled to be able to reset the state, but don't think it blocks getting this merged

**Pull request merged**